action/index.js

```diff
@@ -0,0 +1,50 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const child_process = require('child_process');
+const fs = require('fs');
```

Comment:
It's usually more common to see `import {} from 'module'` rather than the requirejs syntax. But this is fine if we don't want to use that syntax.

Reply:
I defer to you on syntax, I've not written real javascript in quite some time and don't know the current patterns.

Reply:
On making this switch, it didn't compile, so I switched it back to the require statements.